content/docs/nav.yml

```
@@ -24,7 +24,7 @@
     - id: handling-events
       title: Handling Events
     - id: conditional-rendering
-      title: Conditional Rendering
+      title: 條件渲染
```

```suggestion
      title: 條件 rendering
```

---

@bugtender, should we keep the original term `Render`?

```suggestion
      title: 條件 Render
```

---

`Render` Yeah, sorry I forgot.😳